src/PostgREST/Private/QueryFragment.hs

```diff
@@ -151,9 +156,6 @@ fromQi t = (if T.null s then mempty else pgFmtIdent s <> ".") <> pgFmtIdent n
     n = qiName t
     s = qiSchema t
 
-emptyOnFalse :: SqlFragment -> Bool -> SqlFragment
-emptyOnFalse val cond = if cond then mempty else val
```

Comment:
Eh - what? This should have been called `emptyOnTrue`, no? Confusing. I can see why you removed that.

The `emptySnippetOnFalse` follows the same - wrong - logic.

I think those changes are out of scope for this PR, though. I suggest to create a small PR to remove them / fix their name and merge it ahead of this one. Otherwise this kind of change would get lost later in the big squashed commit of this PR...

WDYT?

Reply:
fixed in #1803